katacomb/scripts/continuum_pipeline.py

```diff
@@ -193,13 +193,20 @@ def main():
     post_process_args(args, katdata)
 
     uvblavg_args, mfimage_args, band = _infer_defaults_from_katdal(katdata)
-
+    # Get frequncies and convert them to MHz
+    freqs = katdata.freqs/1e6
     # Get config defaults for uvblavg and mfimage and merge user supplied ones
-    uvblavg_parm_file = pjoin(CONFIG, f'uvblavg_MKAT_{band}.yaml')
-    log.info('UVBlAvg parameter file for %s-band: %s', band, uvblavg_parm_file)
-    mfimage_parm_file = pjoin(CONFIG, f'mfimage_MKAT_{band}.yaml')
-    log.info('MFImage parameter file for %s-band: %s', band, mfimage_parm_file)
-
+    # Check if the observation is narrowband by checking the bandwith size
+    if (freqs[-1] - freqs[0]) < 200:
+        uvblavg_parm_file = pjoin(CONFIG, f'uvblavg_MKAT_narrow_{band}.yaml')
+        log.info('UVBlAvg parameter file for %s-band: %s', band, uvblavg_parm_file)
+        mfimage_parm_file = pjoin(CONFIG, f'mfimage_MKAT_narrow_{band}.yaml')
+        log.info('MFImage parameter file for %s-band: %s', band, mfimage_parm_file)
```

Comment:
I see that you have modified the `continuum_pipeline.py` script to use the updated parameters and to use the correct number of 'nif', but you have left the `continuum_image.py` more or less untouched. 

While I appreciate that for the latter script the user can put the correct inputs in manually, I still think it would be good to modify this script so it uses the correct narrowband parameters by default when running on a narrowband file. Otherwise users might get an unexpected surprise at the end of their imaging run.

Reply:
I have included the parameter files for the offline and modified the script to use correct narrow band parameters.